### PR TITLE
improved elasticsearch debugging

### PR DESCRIPTION
**Pull request by missinglink**
display more information about the elasticsearch request when in debug mode.

before:
```
      {
        "controller:search": {
          "queryType": {
            "autocomplete": {
              "es_result_count": 20
            }
          }
        }
      },
```

after:
```
      {
        "controller:search": {
          "queryType": {
            "autocomplete": {
              "es_took": 37,
              "response_time": 370,
              "retries": 0,
              "es_hits": 46359,
              "es_result_count": 0
            }
          }
        }
      },
```

### CODE REVIEW

File: controller/search.js [hunk 1]
```diff
@@ -102,11 +102,15 @@ function setup( apiConfig, esclient, query, should_execute ){
           const messageParts = [
```
Comment (orangejulius) [inline controller/search.js]: Oh, is `messageParts` actually unused here? This looks like our old (pre-JSON) logging format that is hard to parse and doesn't really do anything useful anymore.

Comment (missinglink) [reply controller/search.js]: Hmm.. good catch, I added a second commit to remove that code

Comment (orangejulius) [reply controller/search.js]: Thanks. Its my fault for leaving it in long ago :)
